demo/lightning/image-segmentation/unet3d.py

```diff
@@ -29,7 +29,7 @@ def __init__(self,
                  weights_init_scale=1.0):
         super(Unet3D, self).__init__()
 
-        filters = [32, 64, 128, 256, 320]
+        filters = [2, 2, 2, 2, 2]
```

Comment:
Do we still want to update this even for the demo version?

Reply:
Thanks for catching. Updated so that filters gets set to smaller values only when run in benchmark mode.